src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceIT.java

```diff
@@ -332,6 +332,45 @@ public void shouldSearchCorrectlyUsingRefundStates() {
                 .body("results[1].transaction_id", is(submittedTransaction.getExternalId()));
     }
 
+    @Test
+    public void shouldSearchForAllExistingRefundsWithDisplaySizeTwo() {
```

Comment:
The test looks good, but looks like searching with a display size is already covered by `shouldSearchUsingAllFieldsAndReturnAllFieldsCorrectly` so I don't think we should add it. Reason being that integration tests are relatively expensive to run, so we should have the minimum possible set to cover functionality.

Reply:
Sure I'll close the PR